cpswt-core/config/build.gradle.kts

```diff
@@ -67,18 +67,18 @@ publishing {
     repositories {
         maven {
             name = "configPublish"
-            val internalRepoUrl = "http://$archivaHostId:$archivaPort/repository/internal"
-            val snapshotsRepoUrl = "http://$archivaHostId:$archivaPort/repository/snapshots"
-            url = uri(if (version.toString().endsWith("SNAPSHOT")) snapshotsRepoUrl else internalRepoUrl)
-
+            val internalRepoUrl = "http://localhost:8080/repository/internal"
```

Comment:
`archivaHostId` and `archivaPort` shouldn't be hardcoded.
If the `gradle.properties` file isn't working, you should be able to provide them on the command-line of `./gradlew` via -PachivaHostId=localhost and -ParchivaPort=8080

Reply:
This comment is fixed and the changes result in a successful build.